frontend/src/components/EvaluationResponses.js

```diff
@@ -1,8 +1,31 @@
-import React, { useState, useMemo, useCallback } from 'react';
+import React, { useState, useEffect, useMemo, useRef } from 'react';
 import { Tab, Row, Tabs } from 'react-bootstrap';
 import styled from 'styled-components';
+import natural from 'natural';
+
 import styles from './EvaluationResponses.module.css';
 import { TextComponent } from './StyledComponents';
+import { evalsColormap } from '../queries/Constants';
+
+// Set up parameters for natural
+const language = 'EN';
+const defaultCategory = 'N';
+const defaultCategoryCapitalized = 'NNP';
+const lexicon = new natural.Lexicon(
```

Comment:
It strikes me that lexical analysis of evaluations is probably something we'd want to split out to a separate file.

Reply:
Hmm, maybe. I'll pull it out in a new commit later if it's necessary. Thanks for the feedback.